view/frontend/web/css/source/_extend.less

```diff
@@ -273,3 +273,46 @@
         }
     }
 }
+
+#payment-method-braintree-cc-vault {
```

Comment:
@paul-gene I know you didnt create this file, but can you wrap the contents in 
`& when (@media-common = true) {` Magento has 2 css files. styles-l and styles-m. If the contents are not wrapped in this query then the styles are added to both files therefor loading twice on the site. 

Drop me a message if you want to know more

Reply:
@chloelangford ah thank you, I was meant to ask someone about that! Yes, I'll sort that out when I do the other fixes mentioned :)

This whole Less file is a bit of a mess and could do with some loving!